src/main/java/dev/vasyl/proj/controller/BookController.java

```diff
@@ -20,33 +23,45 @@
 @RequiredArgsConstructor
 @RestController
 @RequestMapping("/api/books")
```

Comment:
add `server.servlet.context-path=/api` to application.properties

```suggestion
@RequestMapping("/books")
```